include/realizations/catchment/Formulation.hpp

```diff
@@ -48,6 +50,35 @@ namespace realization {
             std::string get_id() const {
                 return this->id;
             }
+
+            /**
+             * Get a header line appropriate for a file made up of entries from this type's implementation of
+             * ``get_output_line_for_timestep``.
+             *
+             * Note that like the output generating function, this line does not include anything for time step.
+             *
+             * @return An appropriate header line for this type.
+             */
+            virtual std::string get_output_header_line(std::string delimiter=",") =0;
```

Comment:
Why not use the macro defined? `DEFAULT_FORMULATION_OUTPUT_DELIMITER`

Reply:
Must have been having a moment there.  Fixed in `2a1184e3d542`